app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+  <title>Recipes</title>
+  <%= stylesheet_link_tag 'application', 'recipes_index' %>
+</head>
+<body>
+  <div class="main-recipes">
+<%= link_to 'New Recipe', new_recipe_path, class: 'new-recipe-link' %>
+<% @recipes.each do |recipe| %>
+  <div class="main-card">
+    <div class="left-column">
+      <h2><%= link_to recipe.name, recipe %></h2>
+      <%= button_to 'Remove', recipe_path(recipe), method: :delete, data: { confirm: 'Are you sure?' }, class: 'remove-button' %>
+    </div>
+    <div class="right-column">
+      <p><%= recipe.description %></p>
+    </div>
+  </div>
+<% end %>
+</div>
+</body>
+</html>
```

Comment:
- [ ] [OPTIONAL] You have done a great job on the recipe index page, however, it will be nice to adjust the deign to look like the provided wireframe. We can see that the `Remove` button does not look user-friendly. Let's adjust the button and the description on each card properly.

### Your Design
<img width="1390" alt="Screenshot 2023-11-18 at 15 36 04" src="https://github.com/lerfast/recipe-app/assets/58692590/ec9445d1-b6c2-4946-8f64-af6a681cbaf8">

### Design Template
<img width="915" alt="Screenshot 2023-11-18 at 15 36 36" src="https://github.com/lerfast/recipe-app/assets/58692590/2422bf24-77fe-455f-b9a0-baae1fc7abe4">